boards/common/blxxxpill/include/periph_conf.h

```diff
@@ -134,33 +134,41 @@ static const timer_conf_t timer_config[] = {
  */
 
 static const qdec_conf_t qdec_config[] = {
+    {
+        .dev        = TIM4,
+        .max        = 0x0000ffff,
+        .rcc_mask   = RCC_APB1ENR_TIM4EN,
+        .chan       = { { .pin = GPIO_PIN(PORT_B, 6),   .cc_chan = 0 },
+                        { .pin = GPIO_PIN(PORT_B, 7),   .cc_chan = 1 } },
+        .bus        = APB1,
+        .irqn       = TIM4_IRQn,
+    },
+    /* this conflicts with PWM */
+#ifndef MODULE_PERIPH_PWM
+    {
+        .dev        = TIM3,
+        .max        = 0x0000ffff,
+        .rcc_mask   = RCC_APB1ENR_TIM3EN,
+        .chan       = { { .pin = GPIO_PIN(PORT_B, 4),   .cc_chan = 0 },
+                        { .pin = GPIO_PIN(PORT_B, 5),   .cc_chan = 1 } },
+        .bus        = APB1,
+        .irqn       = TIM3_IRQn,
+        /* be default TIM3 is routed to PA6 (cc_chan 0) and PA7 (cc_chan 1) */
+        .remap      = AFIO_MAPR_TIM3_REMAP_1,
+    },
+#endif
```

Comment:
This one now works with support for remapping implemented. Still, the QDEC for TIM4 doesn't work. I think it never worked... I wonder how it got upstreamed.

Reply:
OK, this took me way longer to figure out than I had hoped for: The STM32F103xxx bluepill I used has no TIM4. With my Blackpill, the TIM4 (PB6 / PB7) QDEC config works just fine.